config/ignite-log4j.xml

```diff
@@ -101,6 +101,16 @@
         <level value="WARN"/>
     </category>
 
+    <category name="org.gridgain.control.springframework">
+        <level value="WARN"/>
+    </category>
+    <category name="org.gridgain.control.glassfish.grizzly">
+        <level value="WARN"/>
+    </category>
+    <category name="org.gridgain.control.jackson">
+        <level value="ERROR"/>
+    </category>
```

Comment:
Why are you changing the log level for Jackson? We haven't done that before.

Reply:
The following code starts acting up because of the relocation:
 ```java
try {
    Class<?> cls = Class.forName("com.fasterxml.jackson.databind.ext.Java7HandlersImpl");
    impl = (Java7Handlers) ClassUtil.createInstance(cls, false);
} catch (Throwable t) {
    // 09-Sep-2019, tatu: Could choose not to log this, but since this is less likely
    //    to miss (than annotations), do it
    java.util.logging.Logger.getLogger(Java7Handlers.class.getName())
        .warning("Unable to load JDK7 types (java.nio.file.Path): no Java7 type support added");
}
```
It affects the ability to serialize nio classes which we don't use.

Reply:
It looks like a road to nowhere. How many reflection problems have you not found? You just mute error and broke serialization for some classes. We can catch this only on release build, but we doesn't have such tests.

Reply:
exclude `com.fasterxml.jackson.databind.ext` package from renaming